include/ofi_util.h

```diff
@@ -541,6 +610,7 @@ typedef int (*ofi_cmap_signal_func)(struct util_ep *ep, void *context,
 
 struct util_cmap_attr {
 	void 				*name;
+	struct util_cmd_queue		*cmd_queue;
```

Comment:
This would be better of as a structure that is initialized, rather than a pointer that we also need to allocate.

Reply:
Ok, boss :) will do that